shell/platform/darwin/macos/framework/Source/FlutterEngineTest.mm

```diff
@@ -790,6 +790,56 @@ - (nonnull NSView*)createWithViewIdentifier:(int64_t)viewId arguments:(nullable
   EXPECT_TRUE(announced);
 }
 
+TEST_F(FlutterEngineTest, RunWithEntrypointUpdatesDisplayConfig) {
+  BOOL updated = FALSE;
```

Comment:
Obj-C style here and elsewhere: `YES` or `NO` not `TRUE` and `FALSE`.

Reply:
Done. This was copy pasted from another test.

Reply:
Will do a quick blast through to catch the stragglers. Thanks for mentioning.